aria/modeling/service_instance.py

```diff
@@ -180,6 +181,16 @@ def validate_capabilities(self):
                 satisfied = False
         return satisfied
 
+    def find_hosts(self):
```

Comment:
more methods on the models which we'll want to remove in a short while :(
i mean, this method isn't relevant for an instantiated service - it's only used for its (and its dependencies) creation. why would it sit as part of the instance?